wp-parsely-3.0/src/class-parsely.php

```diff
@@ -186,7 +186,7 @@ public function insert_page_header_metadata(): void {
 
 		// Assign default values for LD+JSON
 		// TODO: Mapping of an install's post types to Parse.ly post types (namely page/post).
-		$parsely_page = $this->construct_parsely_metadata( $parsely_options, $post );
+		$parsely_page = $this->construct_parsely_metadata( $parsely_options, get_post( $post ) );
```

Comment:
Another possibility that might be more robust, revert this change:

```suggestion
		$parsely_page = $this->construct_parsely_metadata( $parsely_options, $post );
```
...& up on L#180, something like:

```
$post = get_post( $GLOBALS['post'] );
if ( ! $post instanceof WP_Post ) {
    return;
}
```

Reply:
Yeah, definitely don't override the global :)